src/gotcha.c

```diff
@@ -149,18 +149,6 @@ GOTCHA_EXPORT enum gotcha_error_t gotcha_wrap(struct gotcha_binding_t* user_bind
       //printf("Writing %p, %zu bytes, page size %d remainder is %lx\n",prot_address,protect_size, page_size,((size_t)BOUNDARY_BEFORE(got,page_size)%page_size));
       int res = gotcha_mprotect((void*)prot_address,protect_size,PROT_READ | PROT_WRITE | PROT_EXEC );
       if(!res){
```

Comment:
This and other genius software writing moves on "David writes C." I'll also clean up the commented printf